helpers.md

```diff
@@ -128,7 +131,7 @@ Laravel includes a variety of global "helper" PHP functions. Many of these funct
 
 </div>
 
-<a name="fluent-strings"></a>
+<a name="fluent-strings-method-list"></a>
```

Comment:
Are we worried that this will break existing links to this header in existing blog posts and what not

Reply:
This was an existing duplicate anchor name, so it will have already been broken to some extent, depending on which of the two "Fluent Strings" headings they were attempting to link to.

One was a h2 and one was a h3, so I changed the h3, because the h2 outranked it (:rofl:). However, this h3 comes first in the document, and it seems like that one "wins" when there are duplicates on the page, at least with Firefox. So I may have been better off changing the h2, although to what I'm not sure.

The TOC on this page is also missing a lot of the h3 links and could probably do with a bit of attention.